git_machete/cmd.py

```diff
@@ -2300,22 +2314,28 @@ def handle_untracked_branch(cli_ctxt: CommandLineContext, new_remote: str, b: st
             f"Push branch {bold(b)} with force-with-lease to {bold(new_remote)}?" + pretty_choices('y', 'N', 'q', 'yq', other_remote_choice),
             f"Pushing branch {bold(b)} with force-with-lease to {bold(new_remote)}..."
         )
-    }
+    }[relation]
 
-    yes_actions: Dict[int, Callable[[], None]] = {
+    override = {
+        IN_SYNC_WITH_REMOTE: None,
+        BEHIND_REMOTE: None,
+        AHEAD_OF_REMOTE: "N" if cli_ctxt.opt_no_push or cli_ctxt.opt_no_push_untracked else None,
```

Comment:
Hmmm should `opt_no_push_untracked` be included here and below? IIUC that's the case where only `opt_no_push` but not `opt_no_push_untracked` should matter? 🤔